FabLab/src/styles/global.css

```
@@ -1,76 +1,84 @@
-  --border:   #2a3347;
-  --white:    #ffffff;
-  --muted:    #7a8599;
-  --teal:     #00c9b1;
-  --red:      #c0392b;
-  --red-dim:  #a93226;
-  --r:        3px;
+@import url('https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;600&family=Barlow+Condensed:wght@700;800;900&family=JetBrains+Mono:wght@400;600&display=swap');
+
+/* ── Fonts locales ── */
+@font-face {
+    font-family: 'RobotoCondensed';
```

Stylelint (17.7.0)

[error] Expected no quotes around "RobotoCondensed" (font-family-name-quotes)

(font-family-name-quotes)